LondonBritishLibrary/orient/BLorient611.xml

```diff
@@ -0,0 +1,249 @@
+                        <altIdentifier><idno>Wright cat. CCXXIV</idno></altIdentifier>
+                        <altIdentifier><idno>Wright 124</idno></altIdentifier>
+                    </msIdentifier>
+<msContents>
+
+<msItem xml:id="ms_i1">
+<locus from="5r"/>
+<title type="complete" ref="LIT1295Dersan">Discourses for the festivals of Archangel Michael</title>
+<textLang xml:lang="gez"></textLang>
+
+<msItem xml:id="ms_i1.1">
+<locus from="5r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyHedar"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="11r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyTahsas"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="16v"/>
+<title type="complete" ref="LIT1295Dersan#HomilyTerr"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="25v"/>
+<title type="complete" ref="LIT1295Dersan#HomilyYakkatit"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="29r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMaggabit"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.6">
+<locus from="43v"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMiyazya"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.7">
+<locus from="48r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyGenbot"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.8">
+<locus from="50r"/>
+<title type="complete" ref="LIT1295Dersan#HomilySane"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.9">
+<locus from="56r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyHamle"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="60r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyNahase"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="63r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyMaskaram"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="66r"/>
+<title type="complete" ref="LIT1295Dersan#HomilyTeqemt"/>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="70r"/>
+<title type="complete" xml:lang="en">The miracles</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="85r"/>
+<title type="incomplete" xml:lang="en" ref="LIT2375Synaxa">Extracts from the Synaxarium regarding the festivals</title>
+<textLang xml:lang="gez"></textLang>
+</msItem>
+
+</msItem>
+
+</msContents>
+
+<physDesc>
+<objectDesc form="Codex">
+  <supportDesc>
+      <support>
+          <material key="parchment"/>
+      </support>
+      <extent>
+          <measure unit="leaf">92</measure>
+          <measure unit="blank">Two unnumbered blank leaves at the end.</measure>
```

Comment:
unit="leaf" type="blank", and I guess the content can be a number only.

Reply:
This information you can copy to <foliation>

Reply:
Yes, it was like that when I started to edit the file, forgot to change it.